src/app/games/epaminondas/tests/PositionalEpaminondasMinimax.spec.ts

```diff
@@ -139,9 +128,41 @@ describe('PositionalEpaminondasMinimax', () => {
             [O, O, O, O, O, O, O, _, O, O, O, O, O, O],
         ];
         const lesserState: EpaminondasState = new EpaminondasState(lesserBoard, 1);
-        RulesUtils.expectSecondStateToBeBetterThanFirstFor(minimax,
+        RulesUtils.expectSecondStateToBeBetterThanFirstFor(heuristic,
                                                            lesserState, MGPOptional.empty(),
                                                            greaterState, MGPOptional.empty(),
                                                            Player.ONE);
     });
 });
+
+describe('PositionalEpaminondasMinimax', () => {
+
+    let minimax: PositionalEpaminondasMinimax;
+    const minimaxOptions: AIDepthLimitOptions = { name: 'Level 1', maxDepth: 1 };
+
+    beforeEach(() => {
+        minimax = new PositionalEpaminondasMinimax();
+    });
+    it('should consider possible capture the best move', () => {
```

Comment:
why test in minimax instead of Euristic ?

Reply:
pour éviter de devoir refactorer plein de tests qui sont des tests d'heuristiques mais fait sur le minimax à coup de `chooseNextMove`, j'suis d'accord qu'à terme c'est préférable de tester l'heuristique directement, mais ça demande deux plateaux et j'ai pas le courage dans ce ticket de repasser sur tous ces tests là :p

Reply:
si on teste chooseNextMove c'est bien minimax :D
Je dirais que c'est mieux de combiner les trois tests (liste de moves, préférence sur une métrique donnée, choix de celui qui a cette métrique dans un exemple concrêt), même si tester sur le minimax est plus "intégration"
Par ailleurs si on test le minimax, faisons p-e quand même un fichier juste pour sa création, histoire qu'on ai pas des test sur minimax(a, b, c) alors que dans le composant on a changé à minimax(d, e, f) :D

Reply:
yep, ok pour tester les minimax, mais j'suis pour qu'on ait pour tout nouveau jeu + jeu retravaillé au MINIMUM les tests de move generator + les tests heuristiques, et éventuellement les tests minimax en bonus plus pour de l'intégration du coup

Reply:
évidemment, d'ailleurs, je suis chaud de ralentir le dev util pour ajouter moultes améliorations peu utiles de minimaxs bien unit-testées, et puis augmenter les séances "combats de minimax sur X" comme on avait fait sur Epaminondas (et un peu Dvonn) !
Haha !

Reply:
on peut toujours entrecouper tickets utiles avec AI-battles et autres joyeusetés ! TPR ?